WindowsServerDocs/administration/windows-commands/sc-delete.md

```diff
@@ -43,11 +43,20 @@ Use **Add or Remove Programs** on **Control Panel** to delete DHCP, DNS, or any
 
 ## Examples
 
+### DOS Examples
+
 To delete the service subkey **NewServ** from the registry on the local computer, type:
 ```
 sc delete newserv
 ```
 
+### Powershell Examples
+
+To delete the service subkey **NewServ** from the registry on the local computer, type:
+```
+sc.exe delete newserv
```

Comment:
Simply changing "sc" to "sc.exe" makes the example suitable for both PowerShell and Command Prompt. There is no need for all the bells and whistles of having two separate examples.